addons/overpressure/XEH_postInit.sqf

```diff
@@ -1,9 +1,13 @@
 #include "script_component.hpp"
 
 ["CBA_settingsInitialized", {
-    TRACE_1("settingsInit eh",GVAR(distanceCoefficient));
-    if (GVAR(distanceCoefficient) <= 0) exitWith {};
 
+    TRACE_1("settingsInit eh",GVAR(backblastDistanceCoefficient));
+    TRACE_1("settingsInit eh",GVAR(overpressureDistanceCoefficient));
+
+    // Both settings disabled, no need to initialize overpressure.
+    if (GVAR(backblastDistanceCoefficient) <= 0 && GVAR(overpressureDistanceCoefficient) <= 0) exitWith {};
```

Comment:
I'm not a fan of this, because if you have it turned at the start it requires you to restart the mission. However, we don't notify players about this requirement. It was an issue before this PR, so not this PR's fault.

Imo we should be removing the 0 checks and have the EH always be added. Otherwise we need to invest a bit more work into making it show that it requires a mission restart when the setting is set to 0 at the start only and otherwise it wouldn't show said message.

Reply:
How would I go on using the EH? I'd like to do things proper.

Reply:
Atm don't do anything related to this, I want other opinions on this before anything is changed.